impl.go

```diff
@@ -284,7 +292,7 @@ var errorInterface = []Func{{
 // funcs returns the set of methods required to implement iface.
 // It is called funcs rather than methods because the
 // function descriptions are functions; there is no receiver.
-func funcs(iface string, srcDir string, comments EmitComments) ([]Func, error) {
+func funcs(iface string, srcDir string, recvPkg string, comments EmitComments) ([]Func, error) {
```

Comment:
Probably time to make this

`func funcs(iface, srcDir, recvPkg string, ...)`

Reply:
Do you mean  `func funcs(iface, srcDir, recvPkg string, comments EmitComments )`?
Or `unc funcs(iface string, srcDir string, recvPkg string, ...interface{} )` ?
Or `func funcs(args FuncsArgs)` (FuncsArgs is struct) ?

Reply:
Sorry, I meant the minimal, aesthetic change:

```go
func funcs(iface, srcDir, recvPkg string, comments EmitComments)
```